pkg/forwarder/forwarder_health_test.go

```diff
@@ -38,6 +38,19 @@ func TestHasValidAPIKey(t *testing.T) {
 	assert.Equal(t, &apiKeyValid, apiKeyStatus.Get("API key ending with key3"))
 }
 
+func TestComputeDomainsURL(t *testing.T) {
+	keysPerDomains := map[string][]string{
+		"https://app.datadoghq.com": {"api_key1"},
+	}
+
+	fh := forwarderHealth{keysPerDomains: keysPerDomains}
+	fh.init()
+
+	_, ok := fh.keysPerAPIEndpoint["https://api.datadoghq.com"]
```

Comment:
We should check that it's the only key in `keysPerAPIEndpoint` and that we haven't added a new endpoint but replace it using `assert.Equal`.